add-ons/prometheus/README.md

```diff
@@ -1,7 +1,10 @@
 # Prometheus Add-on
 An addon that will deploy Prometheus.
 
-NOTE: Requires Origin >= 3.6.0-rc.0
+## Prerequisites
+
+- Origin version >= 3.6.0-rc.0
+- [`admin-user`](https://github.com/minishift/minishift/tree/master/addons/admin-user) add-on should be available
 
 Verify you have installed these addons, by following the [general readme](../../README.adoc#download-and-use-community-add-ons).
```

Comment:
"Verify you have installed these addons" - these? This would imply more than one add-on, but afaiu it is just the _admin-user_ add-on which is required, right?

Reply:
+1. Although, maybe better state that in order to access the prometheus web-ui you need to be cluster-admin user, and that that addon is a way to having that user.